### PR TITLE
Status update 2020-12-18

### PR DESCRIPTION
**Pull request by sheenaajay**
Request for a review

### CODE REVIEW

File: _posts/2020-12-18-status-update.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+  - dsc
+  - windows
+  - cloud-ci
+  - puppet-module-gem
+  - rubocop
+  - infrastructure
+  - team
+  - status
+tags:
+---
```
Comment (DavidS) [inline _posts/2020-12-18-status-update.md:17]:
```suggestion
categories:
  - team
  - status
tags:
  - community
  - releases
  - dsc
  - windows
  - cloud-ci
  - puppet-module-gem
  - rubocop
  - infrastructure
---
```

categories are used for constructing the URL, so should only ever be [team, status] for status updates.

File: _posts/2020-12-18-status-update.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+  - releases
+  - dsc
+  - windows
+  - cloud-ci
+  - puppet-module-gem
+  - rubocop
+  - infrastructure
+  - team
+  - status
+tags:
```
Comment (sanfrancrisko) [inline _posts/2020-12-18-status-update.md:16]: Might want to occupy the `tags`

**Review by david22swan — APPROVED** [review]

File: _posts/2020-12-18-status-update.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+---
+
+## Christmas edition
+Puppet is getting ready for the Virtual Chirstmas Party.
+Santa, Food, Drinks, Sweets, Gaming, Charades and many more.
+
+The holiday season upon us, the IAC team would like to wish all of Puppets opensource community Happy Holidays and thank everyone for their contributions through the very challenging year of 2020.
```
Comment (sanfrancrisko) [inline _posts/2020-12-18-status-update.md:23]:
```suggestion
The holiday season upon us, the IAC team would like to wish all of Puppet's Open Source Community Happy Holidays and thank everyone for their contributions through the very challenging year of 2020.
```

File: _posts/2020-12-18-status-update.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+The IAC team will be taking a well-earned break to recharge the batteries.
```
Comment (sanfrancrisko) [inline _posts/2020-12-18-status-update.md:24]:
```suggestion
The IAC team will be taking a well earned break to recharge the batteries.
```

File: _posts/2020-12-18-status-update.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+We look forward to the continued open-source communities help and contributions in 2021.
```
Comment (sanfrancrisko) [inline _posts/2020-12-18-status-update.md:25]:
```suggestion
We look forward to the continued open source communities help and contributions in 2021.
```

**Review by sanfrancrisko — CHANGES_REQUESTED** [review]
Well done @sheenaajay - a lot to incorporate this week! Just a few minor comments

File: _posts/2020-12-18-status-update.md [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+## Content Cloud CI Project
+This week we have 32 of the IAC supported modules running nightlies and part of the PR testing on the Cloud CI services.
+We have worked on a couple of stability issues and in the past 24 hours, the nightlies ran 427 jobs and encountered a total of 15 failures.
+We also improved the provision error tracking. Out of those 15 failures none seem to be related to provisioning.
+Over the last 7 days, we created 5888 VMs for testing purposes compared to 4613, the week before that.
+We will continue to improve the service and the reporting early next year.
+
+## DSC
+
+DSC we are gathering momentum. Approximately 51 new DSC modules on the Forge. With every new module, we learn more and harden the solution.
```
Comment (DavidS) [inline _posts/2020-12-18-status-update.md:107]:
```suggestion
DSC is gathering momentum. We're now up to 51 [new DSC modules](https://forge.puppet.com/dsc) on the Forge. With every new module, we learn more and harden the solution.
```

File: _posts/2020-12-18-status-update.md (hunk 4 above)
Comment (DavidS) [inline _posts/2020-12-18-status-update.md:24]:
```suggestion
The IAC team will be taking a well-earned break to recharge the batteries, and we hope you can too.
```

**Review by DavidS — CHANGES_REQUESTED** [review]

File: _posts/2020-12-18-status-update.md (hunk 1 above)
Comment (sheenaajay) [reply _posts/2020-12-18-status-update.md:17]: Thank you

File: _posts/2020-12-18-status-update.md (hunk 4 above)
Comment (sheenaajay) [reply _posts/2020-12-18-status-update.md]: Got error while adding commits on the same line. Pushed those changes now.

**Comment by sheenaajay** [comment]
Thanks @DavidS @sanfrancrisko for the reviews.

**Pull request merged**